project/TravisOutput.scala

```diff
@@ -0,0 +1,66 @@
+package scala.build
+
+import java.util.TimerTask
+import java.io._
+import java.util.concurrent.TimeUnit
+
+// Prints some output to console periodically if the build itself is silent to avoid Travis CI's 10 minute timeout
+// from aborting the build.
+//
+// A hopefully better behaved version of travis_wait and travis_wait_enhanced.
+object TravisOutput {
+  def install(): Unit = if (saved.getClass.getName != classOf[Redirecter].getName) {
```

Comment:
Maybe check `System.out` here instead of `saved`, and then assign to `saved`? It's not 100% obvious to me that `saved` and `System.out` would always be the same when calling `install`.